packages/ui/src/components/va-avatar/hooks/useAvatarProps.ts

```diff
@@ -0,0 +1,7 @@
+export const useAvatarProps = {
+  color: { type: String, default: 'info' },
+  fallbackSrc: { type: String, default: null },
```

Comment:
As i remember, we have decided not to use `null` for String default.

Reply:
Forget about it. Will fix.